app/api/helpers/permissions.py

```diff
@@ -87,5 +87,113 @@ def decorated_function(*args, **kwargs):
 
     return decorated_function
 
+@second_order_decorator(jwt_required)
+def is_organizer(f):
+    """
+    Allows only Organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+        if kwargs['event_id'] and user.is_organizer(kwargs['event_id']):
+            return f(*args, **kwargs)
+        return ForbiddenError({'source': ''}, 'Access Forbidden').respond()
+
+    return decorated_function
+
+@second_order_decorator(jwt_required)
+def is_coorganizer(f):
+    """
+    Allows Organizer and Co-organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+        if kwargs['event_id']:
+            event_id = kwargs['event_id']
+            if user.is_coorganizer(event_id) and user.is_organizer(event_id):
```

Comment:
Can you please explain this? Why does an user need to be both a co-organizer and organizer of an event to be a co-organizer? co-organizer and organizer are two completely different things and there are some functionalities that are allowed only to organizer but not co-organizer. Also, even if do want to stick with this logic, shouldn't this be `or` instead of `and`?